semconv/internal/v2/http.go

```diff
@@ -0,0 +1,380 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal // import "go.opentelemetry.io/otel/semconv/internal/v2"
```

Comment:
Is it safe to have this package path end in `/v2`?  I think some tools in the toolchain may use that as a signal to look for v2+ module versions.

Reply:
This was intended to be a v2 version of `go.opentelemetry.io/otel/semconv/internal`. That way it doesn't mess with anything that already uses `go.opentelemetry.io/otel/semconv`, but is still has the `internal` package name.

I'm not aware of any tooling that this won't play nice with. Do you have an example?

Reply:
I haven't gone through all the release commands, but from my local testing all our other `make` commands work as expected.